polylogue/cli.py

```
@@ -488,62 +644,121 @@ def run_sync_cli(args: argparse.Namespace, env: CommandEnv, json_output: bool) -
     ui.summary("Sync", lines)
 
 
-def run_status_cli(env: CommandEnv) -> None:
-    result = status_command(env)
+def run_status_cli(args: argparse.Namespace, env: CommandEnv) -> None:
```

Ruff (0.15.9)

[warning] Too many statements (72 > 50)

(PLR0915)